.travis.yml

```diff
@@ -8,16 +8,19 @@ install:
 jobs:
   include:
     - name: "Build, Test, and Run Coverage"
-      script: >
-        cd src
-        &&
-        bazel coverage //... 
-        --compilation_mode=fastbuild 
-        --verbose_test_summary 
-        --instrumentation_filter=//... 
-        --test_output=errors
-        &&
-        bash <(curl -s https://codecov.io/bash) -s bazel-testlogs/
+      script:
+        - cd src
+        # Build, test, run coverage
+        - >
+          bazel coverage //... 
+          --compilation_mode=fastbuild 
+          --verbose_test_summary 
+          --instrumentation_filter=//... 
+          --test_output=errors
+        # Build our CPU-specific targets
+        - bazel build //firmware_new/... --cpu=stm32h7
+        # Upload coverage results
+        - bash <(curl -s https://codecov.io/bash) -s bazel-testlogs/
```

Comment:
Will CI do code coverage for firmware?

Reply:
Great question! So it won't do coverage for any targets that are specific to the stm32h7 CPU, but the normal `bazel build //...` above will get coverage for any targets we can build with clang. Generally the stm32h7 files are hardware-specific stuff that's hard to test anyhow, so I don't think it's a big deal.